posthog/demo/web_data_generator.py

```diff
@@ -155,6 +157,24 @@ def populate_person_events(self, person: Person, distinct_id: str, index: int):
                     timestamp=now() - relativedelta(days=start_day) + relativedelta(seconds=60),
                 )
 
+    def populate_session_recording(self, person: Person, distinct_id: str, index: int):
+        if index != 0:
+            return
+
+        date = now()
+        start_time = self.demo_recording["result"]["snapshots"][0]["timestamp"]
```

Comment:
Just want to make sure the recording will appear on a session today? Because the timestamp of each event would be static, right? Lgtm otherwise

Reply:
It will. I'm abusing repetition here - session recording snapshots have their own timestamp system (due to rrweb) and I'm using that to figure out what time to put onto every SessionRecordingEvent